packages/sdk/src/RequestController.ts

```diff
@@ -17,11 +18,16 @@ export class RequestController {
 	private basePath: string
 	private headers: Record<string, any>
 	private request: AxiosInstance
+	private resultHeaders: ResultHeaders
```

Comment:
I do not like naming conventions. We mix request/response/result. Another thing is that we save only headers that are related to API usage (not all of them). So, maybe we should change the naming from `resultHeaders` to `recentApiLimits` or anything like that? 🤔